app/service-cli-bytearray.cpp

```diff
@@ -27,8 +27,14 @@ int main(int argc, char *argv[]) {
   std::string input = std_input.str();
   using marian::bergamot::Response;
 
+  marian::Ptr<marian::Options> responseOptions = marian::New<marian::Options>();
```

Comment:
Why are we using some weird Options object for what should be a struct?

Reply:
This is converted into a struct. Resolving here.